test/global.test.ts

```diff
@@ -5,12 +5,18 @@
 
 import { IHookCallbackContext } from 'mocha';
 import * as vscode from 'vscode';
+import { TestOutputChannel } from 'vscode-azureextensiondev';
 
-// tslint:disable-next-line:strict-boolean-expressions export-name
+import { ext } from '../extension.bundle';
+
+// tslint:disable-next-line:strict-boolean-expressions
 export let longRunningTestsEnabled: boolean = !/^(false|0)?$/i.test(process.env.ENABLE_LONG_RUNNING_TESTS || '');
+// tslint:disable-next-line:strict-boolean-expressions
+export let protectDefaultResources: boolean = !/^(false|0)?$/i.test(process.env.PROTECT_DEFAULT_RESOURCES || '');
```

Comment:
Lol "PROTECT" sounds so intense

Reply:
So if you don't opt in, then it will delete default resource groups?  That doesn't sound like a good idea...?

Reply:
I'm much more worried about nightly tests leaving extra resources around. I don't think people run these test locally very often and they shouldn't be running Azure tests (which you have to opt in to since they're long running) on a subscription they care about

Reply:
I disagree. I tried to run it yesterday to test out webpack. I certainly wasn't expecting it to delete anything in my subscription.

This should be opt in and set in the build parameters.  (Although really it shouldn't be creating anything in default resource groups in the first place - don't know the reason for that.)